packages/feedback/src/util/handleFeedbackSubmit.ts

```diff
@@ -22,21 +25,15 @@ export async function handleFeedbackSubmit(
     dialog.showError('There was a problem submitting feedback, please wait and try again.');
   };
 
+  dialog.hideError();
+
   try {
-    dialog.hideError();
     const resp = await sendFeedback(feedback, options);
 
-    if (!resp) {
-      // Errored... re-enable submit button
-      showFetchError();
-      return false;
-    }
-
     // Success!
     return resp;
-  } catch {
-    // Errored... re-enable submit button
+  } catch (err) {
+    logger.error(err);
```

Comment:
```suggestion
    __DEBUG_BUILD_ && logger.error(err);
```

for tree shaking ;)

Reply:
Oops, will update this, but will make new PR for the rest.